README.Rmd

```diff
@@ -238,31 +238,108 @@ extract_eq(mod2, wrap = TRUE, terms_per_line = 3,
 
 ## Beyond `lm()`
 
-You're not limited to just `lm` models! Try out logistic regression with `glm()`:
+You're not limited to just `lm` models! **equatiomatic** supports many other models, including logistic regression, probit regression, and ordered logistic regression (with `MASS::polr()`).
+
+### Logistic regression with `glm()`
```

Comment:
Thank you for all of the README updates! I was just talking to Josh about getting a vignette together. The README really has most of it at this point.